src/controllers/accommodationController.js

```diff
@@ -106,5 +107,20 @@ const Accommodation = {
     }
   },
 
+  async bookmarkAccommodation(req, res) {
+    const { userId } = req.userData;
+    const { accommodationId } = req.params;
+
+    const bookmarkData = { UserId: userId, AccommodationId: accommodationId };
+    const result = await BookmarkService.bookmark(bookmarkData);
+    const status = result === 'added' ? 201 : 200;
+    return sendResult(res, status, `Bookmark ${result} successfuly`);
```

Comment:
`Successfully` not `successfuly` this is a typo!

Reply:
okey